app/assets/javascripts/modules/thumb_slider.js

```diff
@@ -37,12 +38,44 @@
       }
     }
 
+    function toggleAriaAttributes() {
```

Comment:
Should we remove this until the next PR that will add this behavior (particularly given the log statements)?